app/controllers/books_controller.rb

```diff
@@ -6,7 +6,7 @@ class BooksController < ApplicationController
   # GET /books
   # GET /books.json
   def index
-    @books = Book.all
+    @books = Book.order(updated_at: 'DESC').page(params[:page]).per(20)
```

Comment:
```suggestion
    @books = Book.order(updated_at: :desc).page(params[:page]).per(20)
```

また、20がマジックナンバーになっているのでkaminariの機能を使って設定するか、定数にしたいですね。

Reply:
orderはシンボルで指定できたんですね。
1ページあたりの表示数はkaminariのデフォルト表示数を書き換えるようにしました。